x-pack/plugins/reporting/server/routes/jobs.js

```diff
@@ -6,7 +6,7 @@
 
 import boom from 'boom';
 import { API_BASE_URL } from '../../common/constants';
-import { jobsQueryFactory } from '../lib/jobs_query';
+import { jobsQueryFactory } from '../lib/perch_jobs_query';
```

Comment:
This might do with a fence, as well
```suggestion
/* BEGIN PERCH CODE */
import { jobsQueryFactory } from '../lib/perch_jobs_query';
/* END PERCH CODE */
```